src/javascripts/file_upload.js

```diff
@@ -0,0 +1,56 @@
+// Show file name, file upload status and check file size
+
+UK_Parliament.fileUploader = function () {
+
+  var fileInput = document.querySelector('[data-file-upload="button"]');
+
+  if (fileInput) {
+
+    // Local variables
+    var
+      fileIcon = document.querySelector('[data-file-upload="icon"]'),
+      fileStatus = document.querySelector('[data-file-upload="status"]');
+
+    fileInput.onchange = function () {
```

Comment:
`document.querySelector` runs on the first element within the document, fails if there are multiple file inputs.